app.js

```diff
@@ -0,0 +1,76 @@
+const http = require('http');
+const app = require('./app/index');
+const automation = require('./scripts/index');
+
+const debug = require('debug')('OG_Yoga:server');
+const server = http.createServer(app);
+
+const port = normalizePort(process.env.PORT || '3000');
+app.set('port', port);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
+server.on('error', onError);
+server.on('listening', onListening);
+
+automation.startScript();
+
+function onError(error) {
```

Comment:
do you plan on writing javadoc for all functions? if so, did you miss one here?

Reply:
Yes, very good comment, I will add the javadoc comment. It is very important to keep the consistency, and javadoc is also important for most functions(especially the ones already being written)